web/about.html

```diff
@@ -326,7 +326,7 @@ <h5 style="position: inline;" class="text-white">Lynn Root</h5>
                 </div>
                 <div class="col-8 align-start">
                   <h5 style="position: inline;" class="text-white">Lisa Marielle Cooper</h5>
-                  <p class="text-white">nan</p>
```

Comment:
Lisa's role appeared as `nan` because it's empty in the csv. I figured it's better if it's just an empty string instead of `nan`.